api/src/devconf.py

```diff
@@ -3,4 +3,8 @@
 """
 
 ERROR_404_HELP = False
-OPENAPI_SPEC = 'openapi/spec.yml'
\ No newline at end of file
+OPENAPI_SPEC = 'openapi/spec.yml'
+
+# SQLalchemy Settings
+SQLALCHEMY_DATABASE_URI = 'sqlite:///:memory:'
```

Comment:
I'm thinking if it would make sens to have a `testconf.py` which uses `'sqlite:///:memory:'`. I'm sure we'll need to develop against a "real" database in our dev environment.

Reply:
A separate test configuration is a sensible idea. 

On the scoping: I am all for isolated tests that leave the DB in a clean state after finishing each test. I tried to incorporate that in the `session` fixture (see above), because I think rolling back after every test is going to be faster than creating and dropping tables after every test case. 

It'd be nice if you could elaborate on how the sad path should look like, since I am not really familiar with that concept.

Reply:
Also: where would you recommend to place the db in the file structure?